router/router.go

```diff
@@ -1079,6 +1079,7 @@ func (worker *workerT) sendRouterResponseCountStat(status *jobsdb.JobStatusT, de
 		"destType":       worker.rt.destName,
 		"respStatusCode": status.ErrorCode,
 		"destination":    destinationTag,
+		"destId":         destination.ID,
```

Comment:
Similar to my comment here:
https://github.com/rudderlabs/rudder-server/pull/2430#discussion_r978345020

As destination is of high granularity, we should exclude it in free multitenant cluster.

Reply:
https://github.com/rudderlabs/rudder-server/pull/2555/files